### ISSUE TITLE
Unhandled Rejection (ReferenceError): registries is not defined

### ISSUE THREAD

**Issue by deepseafishing** [ensdomains/ensjs-v2#55]
I'm having this error 

Unhandled Rejection (ReferenceError): registries is not defined

which deals with the code in line 1027 in the index.js file

**Comment by haltman-at** [comment]
Yup, I'm having this problem too.  I think this is worth expanding on--

You'll get this error if you try to create a new `ENS` without specifying the registry address.  In this case, it would seem that `ENS` is supposed to look up the default registry address based on the network ID you passed in.  Instead, however, you hit an undefined variable error, because the `registries` table was never defined.  It would appear that this case was never tested, I guess...?

Of course, it's still possible to use `ENS` due to the `getENSAddress` function, but this functionality of not having to use this, and just being able to directly specify a network ID to the constructor, which was clearly intended, ought to work.  So, I'm hoping you can fix this fairly quickly.  Thank you!

**Comment by timhc22** [comment]
Also having this issue!

**Comment by Elyx0** [comment]
Same problem here.

**Comment by eggplantzzz** [comment]
+1
